Makefile

```diff
@@ -1,6 +1,6 @@
 # Binaries.
-browserify = ./node_modules/.bin/browserify
-mocha = ./node_modules/.bin/mocha
+browserify = ./node_modules/browserify/bin/cmd.js
+mocha = ./node_modules/mocha/bin/mocha
```

Comment:
Did these not work as-is? I'd rather keep them in the `/.bin` folder in case the packages ever decide to change the filename of their binary